fakestorage/response.go

```diff
@@ -4,7 +4,14 @@
 
 package fakestorage
 
-import "sort"
+import (
+	"net/http"
+	"sort"
+
+	cloudstorage "cloud.google.com/go/storage"
```

Comment:
```suggestion
	"cloud.google.com/go/storage"
```

Reply:
this conflicts with "google.golang.org/api/storage/v1"